go.mod

```diff
@@ -3,10 +3,13 @@ module github.com/SOMAS2020/SOMAS2020
 go 1.15
 
 require (
+	github.com/aclements/go-moremath v0.0.0-20201123183122-222cfcba2589
```

Comment:
yarrrrr
![image](https://user-images.githubusercontent.com/33488131/104108724-d7a34f00-52be-11eb-8ceb-b0a80d544b50.png)

use at your own risk!

Reply:
It's been so useful tho :') 😭

Reply:
this should be fine bc we have `go.sum` to make sure the version matches. I wouldn't worry but eh, the deadline is in one week and no one's hand is going to be itchy enough to actually update the deps.

Reply:
Thank you 💕 Will sort out if anything goes wrong ofc.